akka-docs/src/main/paradox/stream/stream-flows-and-basics.md

```diff
@@ -364,8 +364,8 @@ In Akka Streams almost all computation operators *preserve input order* of eleme
 "cause" outputs `{OA1,OA2,...,OAk}` and inputs `{IB1,IB2,...,IBm}` "cause" outputs `{OB1,OB2,...,OBl}` and all of
 `IAi` happened before all `IBi` then `OAi` happens before `OBi`.
 
-This property is even upheld by async operations such as `mapAsync`, however an unordered version exists
-called `mapAsyncUnordered` which does not preserve this ordering.
+This property is even upheld by async operations such as @apidoc[mapAsync](stream.*.Source) { scala="#mapAsync[T](parallelism:Int)(f:Out=%3Escala.concurrent.Future[T]):FlowOps.this.Repr[T]" java="#mapAsync(int,akka.japi.function.Function)" }, however an unordered version exists
+called @apidoc[mapAsyncUnordered](stream.*.Source) { scala="#mapAsyncUnordered[T](parallelism:Int)(f:Out=&gt;scala.concurrent.Future[T]):FlowOps.this.Repr[T]" java="#mapAsyncUnordered(int,akka.japi.function.Function)" } which does not preserve this ordering.
 
 However, in the case of Junctions which handle multiple input streams (e.g. `Merge`) the output order is,
```

Comment:
Should I add links to `Merge`, `Zip`, `MergePreffered` and `MergePrioritized`? Which links? I didn't find any javadsl alternatives. Only objects.

Reply:
Ok, then let's skip it. Thank you.